camelot/admin/icon.py

```diff
@@ -0,0 +1,32 @@
+"""Serializable icon for building admins"""
+
+from dataclasses import dataclass
+
+from camelot.core.serializable import DataclassSerializable
+
+@dataclass
+class Icon(DataclassSerializable):
+    """An icon that can be used to build admins
+
+.. attribute:: name
+
+    The name of the icon. This can be a front awesome name or image name ending with ".png".
```

Comment:
wat we hier ook kunnen doen is als name een 'qrc' toelaten, dwz een qt resource

we kunnen in ieder geval geen 'vfinance' stuff injecteren...